src/widget/wvumetergl.h

```diff
@@ -38,8 +38,9 @@ class WVuMeterGL : public QGLWidget, public WBaseWidget {
     void showEvent(QShowEvent* /*unused*/) override;
     void setPeak(double parameter);
 
-    // To make sure we render at least once even when we have no signal
-    bool m_bHasRendered;
+    // To make sure we render at least N times even when we have no signal,
+    // for example after showEvent()
+    int m_iNeedsRender;
```

Comment:
The variable name can be improved, sounds like a boolean. We have no rule for an i prefix in Mixxx. 
How about m_pendingRendings

It looks like it is a workaround for a first failing rendering. Is it reliable that the first fails and the second succeeds? 
Is there any measure we can use to detect a failing attempt? This would allow to go back to the boolean and might be more reliable. 

On the other hand there is no harm with this approach if it has been proven to be a solution, except the extra CPU for a pointless first render.

Reply:
> It looks like it is a workaround for a first failing rendering.

I noticed "// 2 pendings renders, in case we have triple buffering" in one of @m0dB's PRs so I tried this. So the first rendering is not pointless but buffered IIUC.
Can we somehow figure if triple-buffering is enabled and use either 1 or 2?

Reply:
I haven't found a way (and honestly, the tripple buffering is an assumption, but it's the only explanation I see for what I have been seeing while debugging my QOpenGLWindow based branch). Anyway, the cost of 1 extra render is surely neglectable, so I don't think it's worth the effort.